src/libraries/System.Net.Primitives/src/System/Net/CredentialCache.cs

```diff
@@ -266,7 +266,7 @@ internal static CredentialEnumerator Create(CredentialCache cache)
             private readonly CredentialCache _cache;
             private readonly int _version;
             private bool _enumerating;
-            private NetworkCredential _current;
+            private NetworkCredential _current = null!;
```

Comment:
Is this being initialized by the constructor? If not, it should be declared nullable.

Reply:
From https://github.com/dotnet/runtime/blob/e041241bd94adca59a6ec90173e1d90599deed87/docs/coding-guidelines/api-guidelines/nullability.md
> DO NOT be concerned with the validity of an enumerator's Current annotations when used before MoveNext has been called or after MoveNext has returned false.

Reply:
I see. I'd rather declare the field as nullable and then just force non-nullability of the property getter.

Reply:
Done.